master/internal/rm/kubernetesrm/pods.go

```diff
@@ -615,13 +602,13 @@ func (p *pods) refreshPodStates(ctx *actor.Context, allocationID model.Allocatio
 			continue
 		}
 		pod := pod
-		p.podStatusCallback(ctx.Self().System(), watch.Event{Object: &pod})
+		p.podStatusCallback(system, watch.Event{Object: &pod})
```

Comment:
Would it be possible to completely remove actors from pods if we can resolve this? Maybe passing in a callback or interface to Initialize?

Reply:
I'm going to fix this when i do k8s resource pool. felt it would be easier then.

Reply:
at that point, i won't be able to ctx.Tell the cluster anyway, so it will have to happen.